Mathlib/Data/Fintype/Shrink.lean

```diff
@@ -0,0 +1,22 @@
+/-
```

Comment:
Isn't this file also in another PR of yours?

Reply:
Yes sorry, I forgot to record the dependency on #13988